src/content/en/fundamentals/media/manipulating/cheatsheet.md

```diff
@@ -191,11 +194,6 @@ remove all whitespace including the final carriage return.
 For the `-key` flag use the key created earlier and stored in the media.key
 file. However, when entering it on the command line, be sure you've removed its
 whitespace. For the `-key_id` flag repeat the key value.
```

Comment:
Not sure what answer you need. I think you can just say:

> key_id is an identifier that uniquely identifies the key.

If you want, you can mention that:

> It is available in packager outputs in plaintext. It will be used by the player / content decryption module to select the right key to decrypt the content.

Let me know if you need anything else.